osu.Game/Graphics/OsuColour.cs

```diff
@@ -383,6 +383,7 @@ public static Color4 ForegroundTextColourFor(Color4 backgroundColour)
 
         // Content Background
         public readonly Color4 B5 = Color4Extensions.FromHex(@"222a28");
+        public readonly Color4 B6 = Color4Extensions.FromHex(@"1c1719"); // hsl(333,10%,10%)
```

Comment:
This is not a thing that should need adding. The colour you want already exists as `Background6` in `OverlayColourProvider` instead, I'm pretty sure.